lib/sda/host_test.go

```diff
@@ -14,6 +14,7 @@ import (
 
 // Test setting up of Host
 func TestHostNew(t *testing.T) {
+	defer dbg.AfterTest(t)
```

Comment:
Yes, problem with `AfterTest` is, that you only know after ALL tests whether a go-routine is hanging around. So if the test fails with a go-routine too much, you don't know in *which* test the go-routine was started. Thus I put it back in.

Best would be to have a `before` and `after` for testing, but that doesn't exist in vanilla go-tests...